src/components/biggive-footer/biggive-footer.tsx

```diff
@@ -170,6 +170,9 @@ export class BiggiveFooter {
                 <li>
                   <a href={makeURL('Blog', this.blogUrlPrefix, 'our-people')}>Our People</a>
                 </li>
+                <li>
+                  <a href={makeURL('Blog', this.blogUrlPrefix, 'donation-funds')}>Donate By Bank</a>
```

Comment:
This copy feels confusing to me - I might guess what it means but I think I'd have to read the page to be sure. Might be better to say "Donate By Bank Transfer"? (and maybe with lowercase 'by' - we use lowercase 'for' in campaign names).